tests/spectron/openACP.spectron.js

```diff
@@ -0,0 +1,57 @@
+const Application = require('./spectronSetup');
+const WebDriver = require('./spectronWebDriver');
+const { isMac } = require('../../js/utils/misc.js');
+const Utils = require('./spectronUtils');
+var app = new Application({
+    startTimeout: 1200000,
+    waitTimeout: 1200000
+});
+const WindowsAction = require('./spectronWindowsActions');
+const WebActions = require('./spectronWebActions');
+const ifc = require('./spectronInterfaces.js');
+const specconst = require('./spectronConstants.js');
+let webActions, windowAction;
+
+describe(' Open ACP inside Electron when clicking on the "Go to AC portal"', () => {
```

Comment:
@tranducanh you can skip the tests for Mac `!isMac ? describe() : skip.describe();`. Instead of launching and closing the app

Reply:
hi Kiran 
with https://github.com/symphonyoss/SymphonyElectron/pull/454  VishwasShashidhar want us to use isMac inside the function. Can you recommend me a better solution in this case